cpp/src/parquet/properties.h

```diff
@@ -532,6 +571,50 @@ class PARQUET_EXPORT WriterProperties {
       return this->disable_statistics(path->ToDotString());
     }
 
+    /// Enable writing bloom filter in general for all columns. Default disabled.
+    ///
+    /// Please check the link below for more details:
+    /// https://github.com/apache/parquet-format/blob/master/BloomFilter.md
+    Builder* enable_bloom_filter() {
+      default_column_properties_.set_bloom_filter_enabled(true);
+      return this;
+    }
+
+    /// Enable bloom filter for the column specified by `path`.
+    /// Default disabled.
+    Builder* enable_bloom_filter(const std::string& path) {
+      auto iter = bloom_filter_options_.find(path);
+      if (iter == bloom_filter_options_.end() || iter->second == std::nullopt) {
+        bloom_filter_options_[path] = BloomFilterOptions();
```

Comment:
I don't understand: you can change the default bloom filter options for all columns in `set_bloom_filter_options`, but if you enable bloom filter for one column, then it gets the default ndv/fpp values above?

Reply:
Yeah, I'll remove this. I think user should **explicit** set the BloomFilter arguments ( like ndv), otherwise he/she will get unexpected result